content/docs/2.12/concepts/admission-webhooks.md

```diff
@@ -15,3 +15,7 @@ KEDA will block all incoming changes to `ScaledObject` that don't match these ru
 - The scaled workload (`scaledobject.spec.scaleTargetRef`) is already autoscaled by another other sources (other ScaledObject or HPA).
 - CPU and/or Memory trigger are used and the scaled workload doesn't have the requests defined. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**
 - CPU and/or Memory trigger are **the only used triggers** and the ScaledObject defines `minReplicaCount:0`. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**
+
+KEDA will block all incoming changes to `TriggerAuthentication`/`ClusterTriggerAuthentication` that don't match these rules:
+
+- The specified identity ID for Azure AD Workload Identity and/or Pod Identity is empty. (Default/unset identity ID will be passed through.)
```

Comment:
Sorry but I still think that it's confusing, too much slashes in the same sentence and I think that this still is confusing :( 
What about:
```suggestion
- The specified identity ID for Azure AD Workload Identity and/or Pod Identity is empty. (Default/unset identity ID will be passed through.)
	> NOTE: This only applies if TriggerAuthentication/ClusterTriggerAuthentication is overriding the default identityId provided to KEDA
```